tests/ns-adapter/tests/main_test.go

```diff
@@ -14,60 +14,74 @@ import (
+	"github.com/kyma-incubator/compass/components/director/pkg/log"
+	"github.com/kyma-incubator/compass/tests/pkg/util"
+
 	"github.com/google/uuid"
 	"github.com/kyma-incubator/compass/components/director/pkg/auth"
 	"github.com/kyma-incubator/compass/components/director/pkg/oauth"
 	"github.com/kyma-incubator/compass/tests/pkg/token"
 	"github.com/stretchr/testify/require"
 
 	"github.com/kyma-incubator/compass/tests/pkg/gql"
-	"github.com/kyma-incubator/compass/tests/pkg/server"
 	"github.com/kyma-incubator/compass/tests/pkg/tenant"
 	"github.com/machinebox/graphql"
-	log "github.com/sirupsen/logrus"
 
 	"github.com/pkg/errors"
 
 	"github.com/vrischmann/envconfig"
 )
 
 type config struct {
-	ExternalServicesMockURL string        `envconfig:"EXTERNAL_SERVICES_MOCK_URL"`
-	ClientID                string        `envconfig:"CLIENT_ID"`
-	ClientSecret            string        `envconfig:"CLIENT_SECRET"`
-	TokenURL                string        `envconfig:"TOKEN_URL,optional"`
-	InstanceURL             string        `envconfig:"INSTANCE_URL,optional"`
-	TokenPath               string        `envconfig:"TOKEN_PATH,optional"`
-	RegisterPath            string        `envconfig:"REGISTER_PATH,optional"`
-	Subaccount              string        `envconfig:"NS_SUBACCOUNT,optional"`
-	CreateClonePattern      string        `envconfig:"CREATE_CLONE_PATTERN,optional"`
-	CreateBindingPattern    string        `envconfig:"CREAT_BINDING_PATTERN,optional"`
-	DefaultTestTenant       string        `envconfig:"DEFAULT_TEST_TENANT"`
-	Domain                  string        `envconfig:"DOMAIN"`
-	DirectorURL             string        `envconfig:"DIRECTOR_URL"`
-	AdapterURL              string        `envconfig:"ADAPTER_URL"`
-	Timeout                 time.Duration `envconfig:"default=60s"`
-	X509Config              oauth.X509Config
-	UseClone                bool `envconfig:"default=false,USE_CLONE"`
+	ExternalServicesMockURL        string        `envconfig:"EXTERNAL_SERVICES_MOCK_URL"`
+	ClientID                       string        `envconfig:"CLIENT_ID"`
+	ClientSecret                   string        `envconfig:"CLIENT_SECRET"`
+	TokenURL                       string        `envconfig:"TOKEN_URL,optional"`
+	InstanceURL                    string        `envconfig:"INSTANCE_URL,optional"`
+	TokenPath                      string        `envconfig:"TOKEN_PATH,optional"`
+	RegisterPath                   string        `envconfig:"REGISTER_PATH,optional"`
+	Subaccount                     string        `envconfig:"NS_SUBACCOUNT,optional"`
+	CreateClonePattern             string        `envconfig:"CREATE_CLONE_PATTERN,optional"`
+	CreateBindingPattern           string        `envconfig:"CREAT_BINDING_PATTERN,optional"`
+	DefaultTestTenant              string        `envconfig:"DEFAULT_TEST_TENANT"`
+	Domain                         string        `envconfig:"DOMAIN"`
+	DirectorURL                    string        `envconfig:"DIRECTOR_URL"`
```

Comment:
Domain and DirectorURL should be removed as they are removed from the test definition